PlayCover/Views/Uninstaller.swift

```diff
@@ -123,6 +123,7 @@ class Uninstaller {
             FileManager.default.delete(at: app.entitlements)
         }
 
+        app.removeAlias()
```

Comment:
The directory `~/Applications/PlayCover` should be added to the `pruneURLs` array and the `removeAlias` should be a new uninstall setting for cases like uninstalling and reinstalling apps.

Reply:
I added the directory to another array since it has to be indexed by display name instead of bundle id.

As for the second point, I have tested updating and reinstalling apps, and the existing aliases will still work for the newly replaced apps. I don't think we should leave an alias for inexistent references (deleted apps), so removing them should be default. The only way an alias can stay if an app is uninstalled, is if the app itself is deleted from Finder and not PlayCover. If that happens, using prune will take care of the orphan references.